skop08-sp2

p-skoda opened the pull request:
Link na aplikaci: https://eso.vse.cz/~skop08/sp2/

p-skoda committed:
skop08-sp2

p-skoda committed:
Version 2

Refactor code, add sweet alert library, fix hours

nvbach91 commented:
- Pekne, nicmene narazil jsem na bug kdyz se podari vlozit neexistujici mesto do historie tak se po kazdem otevirani aplikace zobrazi error, a nejde se toho zbavit

![image](https://github.com/user-attachments/assets/895348a5-cf4c-4f64-9c7f-2394a8ea5a7a)

- bylo by dobre pridat moznost konverze teploty do jinych metrickych systemu
- nemelo by se vybirat stejny element vicekrat v prubehu programu
- soubory davejte do spravnych slozek

p-skoda committed:
Add files via upload

Oprava dle těchto poznamek:
-bug kdyz se podari vlozit neexistujici mesto do historie tak se po kazdem otevirani aplikace zobrazi error, a nejde se toho zbavit
-bylo by dobre pridat moznost konverze teploty do jinych metrickych systemu
-nemelo by se vybirat stejny element vicekrat v prubehu programu
-soubory davejte do spravnych slozek

p-skoda committed:
version 3

Oprava NaN u min a max teplot

petr-skoda committed:
Delete www/skop08/sp2/main.js

Clean repository, delete wrong structure

petr-skoda committed:
Delete www/skop08/sp2/styles.css

Clean repository, delete wrong structure file

p-skoda commented:
Byly upraveny všechny problémy, které byly popsány v předchozím komentáři. Plus jsem uklidil repository ve kterém byla špatná struktura souborů

nvbach91 commented:
seznam vyhledavani neradit
duplicitni misto?
oblibene mista
metricky system ukladat

p-skoda committed:
Upravy ze zkousky

Upraveno následující:

seznam vyhledavani neradit
oblibene mista
metricky system ukladat

p-skoda commented:
Bylo upraveno následující:

seznam vyhledavani neradit
oblibene mista
metricky system ukladat
